ROADMAP.md

```diff
@@ -1,56 +1,47 @@
-# KF Serving 2019 Roadmap
-## Q3 2019
-### v0.2 Integrate with the ML Ecosystem (ETA: August 15, 2019)
-Objective: "Continue to simplify the user experience by deeply integrating with the Kubeflow Ecosystem."
-* Kubeflow Integration
-    * Prepare KFServing to release v0.2 and v0.3 alongside Kubeflow v0.7.
-    * Integrate with `kfctl generate` and `kfctl apply`.
-    * Deploy as a [Kubernetes Application](https://github.com/kubernetes-sigs/application).
-    * Integrate with Kubeflow Pipelines to enable model deployment from a Pipeline.
-    * Integrate with Fairing to enable model deployment from a Notebook.
-    * Achieve 20% End-to-End Test Coverage of Supported Features. (See v0.3 for 80%).
-    * Support PVCs to enable integration with on-prem Kubeflow installations.
-    * Document Installation for various cloud providers (GCP, IBM Cloud, Azure, AWS).
+# KF Serving Roadmap
+## 2019 - 2020
 
-Objective: "Empower users to deeply understand their predictions and validate KFServing's static graph architecture."
-* Explainability
-    * Deploy a predictor and explainer, powered by Alibi.
-    * Deploy a predictor and explainer, powered by user specified explainer container.
+### v0.4 Performance (ETA: Jan 31, 2019)
+Objective: "Prevent performance regressions across a known set of representative models."
+* Automated Performance Tests 
+    * Define a set of Models to test covering a wide array of usecases and frameworks.
+    * Publish performance results over time to enable regression tracking.
 
-Objective: "Increase coverage of ML frameworks to support previously unsupported customer workloads."
-* Frameworks
-    * Deploy a ONNX model
-    * Explore supporting other model serialization mechanisms for certain frameworks (e.g. saving PyTorch models with dill)
+Objective: "Enable users to deploy latency sensitive models with KFServing."
+* High Performance Dataplane
+    * Enable support for GRPC or similar.
+    * Continue to support existing HTTP Dataplane.
```

Comment:
Do we have a plan to support enable caching of model files? It's really frustrating when pod fail due to downloading model files. Especially considering knative will autoscale up/down the instances. It will significantly speed up the time before autoscaled service be ready too.